runtime/common/src/lib.rs

```diff
@@ -59,6 +62,15 @@ pub use pallet_timestamp::Call as TimestampCall;
 #[cfg(any(feature = "std", test))]
 pub use sp_runtime::BuildStorage;
 
+/// A parent blockchain identifier. Used primarily when parachains need to refer to a parent chain's
+/// `AccountId`.
+#[derive(Clone, Eq, PartialEq, Encode, Decode, scale_info::TypeInfo)]
+pub struct ParentChainId(pub BoundedVec<u8, ConstU32<20>>);
```

Comment:
unclear to me why this is always a 20 byte identifier?

Reply:
`PalletId` also supports some fixed number of bytes.

Reply:
But I don't like that we need to have some unchecked call or similar here. This also doesn't really make any sense to use a bounded vec here?